app/views/api/v1/application_mailer/target_match.html.erb

```diff
@@ -0,0 +1,13 @@
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <h1><%= @user.email %> your target <%= @target.title %> just got matched!</h1>
+    <p>
+      You can now have a conversation with the matched user.
+      Remember to be polite and hava a good time!
```

Comment:
Typo:
```suggestion
      Remember to be polite and have a good time!
```

And also, this can be moved to locales file. Not a blocker, just a suggestion.